src/pytest_plugins/consume/hive_simulators/conftest.py

```diff
@@ -40,6 +42,16 @@ def pytest_addoption(parser):
         default=False,
         help="Log the timing data for each test case execution.",
     )
+    consume_group.addoption(
+        "--strict-exception-matching",
```

Comment:
what is the difference if it is not set to strict?

Reply:
Yeah, I changed this to `--disable-strict-exception-matching` for it to be enabled by default and in case we need to turn it off for some client we can do so, but ideally we should simply fix the exception messages if that's the case.

Reply:
Merged that one :)